RELEASE_NOTES.md

```diff
@@ -6,47 +6,28 @@ Example Note:
 ## Example Heading
 Details about the thing that changed that needs to get included in the Release Notes in markdown.
 -->
-# knife-ec2 0.12.0 release notes:
+# knife-ec2 0.13.0 release notes:
 
-This release of `knife-ec2` adds features like spot pricing, EBS volume encryption support and some bug fixes. There is also a dependency update for the `fog` gem and 'em-winrm' dependency is removed.
+This release of `knife-ec2` adds feature to bootstrap the `Windows` instance without the need for users to provide the `user-data`. Also, it adds support for users to pass `AWS config file` option on the `CLI` containing the `AWS configurations` to read the config like `region` information.
```

Comment:
adds feature to bootstrap Windows instances over SSL without the need for users to provide the `user-data`.